tests/test_schedule.py

```diff
@@ -132,7 +132,7 @@ def test_periodic_command(self):
         sched = schedule.InvokeScheduler()
         target = mock.MagicMock()
 
-        before = datetime.datetime.utcnow()
+        before = datetime.datetime.now()
```

Comment:
This change causes the tests to fail on my machine in UTC-4, but it works if I specify a time zone.

```suggestion
        before = datetime.datetime.now(tz=datetime.timezone.utc)
```